### ISSUE TITLE
microG stub package

### ISSUE THREAD

**Issue by billie80** [microg/GmsCore#1867]
This is somewhat related to issue #1745.

The developers of ShiftOS would consider shipping microG as a system app. However, they are concerned that delivering it as a default app might be problematic for users. Some may want a system that is completely free of Google services, although it is possible to disable device registration, cloud messaging, etc. in microG.

So the idea came up to ship an empty signed stub package. That way, anyone who wants to use it could install the fully functional version of microG as an update.

A non-functional stub would certainly be more acceptable for those who wont use it and everyone else could use it as a system app. It would also support the adoption of microG.

What do you think? Would that be possible?

**Comment by ale5000-git** [comment]
@mar-v-in
This will also help when the system partition is so small that microG GmsCore doesn't fit.

**Comment by Pdevo** [comment]
This would be a very nice and smart addition. Since you are reasonably going to update microG over the time, you end up having the original full package (and libs too) in /system partition. Of course, not for everyone this is an issue, but for older smartphones some extra space saved can be nice :)

**Comment by rokejulianlockhart** [comment]
I disagree, because this causes situations like https://www.reddit.com/u/BEEDELLROKEJULIANLOC/comments/1087vs5/comment/j7i7sl4/?utm_source=share&utm_medium=web2x&context=3

**Comment by billie80** [comment]
Not related, as the stub is currently not a thing!

**Comment by billie80** [comment]
Also note that this is completely optional to implement. Shift provides a flavor with GMS and without. I see no reason to install without GMS and then add GMS again.
